src/BlockTypes/FeaturedItem.php

```diff
@@ -0,0 +1,339 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * FeaturedItem class.
+ */
+abstract class FeaturedItem extends AbstractDynamicBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name;
+
+	/**
+	 * Default attribute values.
+	 *
+	 * @var array
+	 */
+	protected $defaults = array(
+		'align' => 'none',
+	);
+
+	/**
+	 * Global style enabled for this block.
+	 *
+	 * @var array
+	 */
+	protected $global_style_wrapper = array(
+		'text_color',
+		'font_size',
+		'border_color',
+		'border_radius',
+		'border_width',
+		'background_color',
+		'text_color',
```

Comment:
It seems that `text_color` is present twice.

Super extreme nitpick of mine: when order is not important, what do you think of ordering stuff in alphabetical order? It makes things easier to find and also to spot duplicates like this.

Reply:
Makes total sense 👌